### ISSUE TITLE
actin crosslink formation  MF in BP ontology

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#19185]
This is defined as an adaptor (i.e a protein linking 2 action filaments)

actin crosslink formation (GO:0051764) -
Type of cytoskeletal adaptor
Definition	Parents
The process in which two or more actin filaments are connected together by proteins that act as crosslinks between the filaments. The crosslinked filaments may be on the same or differing axes.

suggest ->  

NTR actin cross linker
As a descendant of GO:0008093    cytoskeletal adaptor activity

Def
The binding activity of a protein that connects together two actin filaments.

**Comment by ValWood** [comment]
note to self check existing annotations
actin filament binding with actin filament bundle assembly
is sometimes crosslinking

**Comment by pgaudet** [comment]
I suppose you need to first create the new term to allow people to move their annotations? 

I think there is a Noctua maintenance outage this week so you could probably obsolete at the end of next week. 

Thanks, Pascale

**Comment by ValWood** [comment]
OK I'll take this one. I'll add the new term first. No big rush.

**Comment by ValWood** [comment]
@dragon-ai-agent please

add new MF term
actin-filament cross-linking activity
An adaptor activity that brings together two actin filaments, enabling the bundling or networking of actin filaments (F-actin). 

As a descendant of GO:0008093 cytoskeletal anchor activity
(please also switch the name and synonym of GO:0008093 cytoskeletal anchor activity, make 
SYNONYM | cytoskeletal adaptor activity the primary label, and GO:0008093 cytoskeletal anchor activity a narrow synonym

**Comment by dragon-ai-agent** [comment]
🤖 Working on it...

Follow along: [View workflow run](https://github.com/geneontology/go-ontology/actions/runs/24507293848)

*— @dragon-ai-agent*

**Comment by pgaudet** [comment]
This seems odd - isn't cross linking covalent ? Adaptor is non-covalent. 

> The binding activity of a molecule that brings together two or more molecules through a selective, ***non-covalent***, often stoichiometric interaction, permitting those molecules to function in a coordinated way.

**Comment by dragon-ai-agent** [comment]
Done — PR geneontology/go-ontology#31911.

**Changes:**

- Added new MF term **GO:7770064 `actin-filament cross-linking activity`** as a child of GO:0008093 (with `has_part` GO:0051015 actin filament binding). Definition used verbatim from your request: *"An adaptor activity that brings together two actin filaments, enabling the bundling or networking of actin filaments (F-actin)."*
- Renamed **GO:0008093**: primary label is now `cytoskeletal adaptor activity`; `cytoskeletal anchor activity` is retained as a NARROW synonym.

You can now migrate the F-actin cross-linking annotations (ain1, fimbrin, and the stg1/myo51 candidates from PMID:39330407) to GO:7770064. Obsoletion of GO:0051764 `actin crosslink formation` is out of scope for this PR — happy to open a separate PR for that once you've finished the annotation migration, just say the word.

Note on definition provenance: I used `GOC:vw` only. I did not add PMID:39330407 to the def xrefs since the wording is yours rather than extracted from the paper, but we can add it if you'd prefer.

One minor naming choice to flag: the label is `actin-filament cross-linking activity` (hyphenated `actin-filament`) rather than the `actin filament cross-linking activity` spelling — this avoids collision with the historical obsolete `GO:0003780 actin cross-linking activity` and keeps the scope clearly filament-to-filament. The unhyphenated form is an EXACT synonym.

---
🤖 **Generated by @dragon-ai-agent**
- Model: `claude-opus-4-6`
- Agent harness: claude-code
- Triggered by: @ValWood
- Run: [View workflow run](https://github.com/geneontology/go-ontology/actions/runs/24507293848)

**Comment by pgaudet** [comment]
I dont see the PMID supporting this (to check whether this is covalent or not)

**Comment by pgaudet** [comment]
Let's not add leaf terms without PMIDs.

**Comment by ValWood** [comment]
Good point. Do you have a better parent. We can use the references on the existing BP term

**Comment by ValWood** [comment]
There is no reference on the original term. Here's a review.
But it appears to be binding, not covalent 
https://pmc.ncbi.nlm.nih.gov/articles/PMC10070769/

**Comment by ValWood** [comment]
checking with Claude:
is actin cross-linking covalent or non-covalent


Actin cross-linking is primarily non-covalent.

Actin filaments (F-actin) are cross-linked by actin-binding proteins (ABPs) — such as filamin, α-actinin, fimbrin/plastin, and fascin — which physically bridge filaments through non-covalent protein–protein interactions. These interactions involve hydrogen bonds, electrostatic forces, and hydrophobic contacts between the ABP and actin.

This non-covalent nature is functionally important because it allows the cytoskeleton to be dynamic and remodelled — filaments can be cross-linked, bundled, or released in response to cellular signals.

There is one notable exception: transglutaminases can catalyse covalent isopeptide bonds between glutamine and lysine residues on actin, creating covalent cross-links. This occurs in specific contexts such as cell death, epidermal cornification, and certain pathological states — **but this is not the typical physiological mechanism**.

So in short: physiological actin cross-linking = non-covalent; transglutaminase-mediated = covalent (specialised/pathological contexts).

**Comment by ValWood** [comment]
@dragon-ai-agent please

add definition xref PMID:37025173

**Comment by dragon-ai-agent** [comment]
🤖 Working on it...

Follow along: [View workflow run](https://github.com/geneontology/go-ontology/actions/runs/24510991370)

*— @dragon-ai-agent*

**Comment by dragon-ai-agent** [comment]
Added `PMID:37025173` to the definition xref of GO:7770064 `actin-filament cross-linking activity`. Pushed to PR geneontology/go-ontology#31911.

Current def line:
```
def: "An adaptor activity that brings together two actin filaments, enabling the bundling or networking of actin filaments (F-actin)." [GOC:vw, PMID:37025173]
```

Validation: SPARQL QC suite passes. Addresses @pgaudet's request for a supporting reference — Lehne & Bogdan 2023 (Front Cell Dev Biol, review) is consistent with the non-covalent adaptor framing.

---
🤖 **Generated by @dragon-ai-agent**
- Model: `claude-opus-4-6`
- Agent harness: claude-code
- Triggered by: @ValWood
- Run: [View workflow run](https://github.com/geneontology/go-ontology/actions/runs/24510991370)

**Comment by ValWood** [comment]
For reannotation:

Classical crosslinkers (two actin-binding domains, directly bridge filaments):
Sac6 (Fimbrin) — yeast fimbrin is encoded by the SAC6 gene and is a cytoskeletal protein that crosslinks F-actin. [PubMed Central](https://pmc.ncbi.nlm.nih.gov/articles/PMC1206575/) It has two actin-binding domains and is the best-characterised yeast actin crosslinker, essential for endocytic patch function.
Scp1 (Transgelin/SM22 homologue) — the S. cerevisiae calponin/transgelin homologue Scp1 functions with fimbrin to regulate stability and organisation of the actin cytoskeleton. [Oxford Academic](https://academic.oup.com/femsre/article/38/2/213/497279) It crosslinks filaments and its loss exacerbates endocytic defects in sac6 mutants.
These two are the core crosslinkers. The yeast fimbrin Sac6 and yeast transgelin Scp1 crosslink actin filaments in vitro and are localised to the actin filament network at endocytic sites. Yeast fimbrin is essential for clathrin-mediated endocytosis, and its absence leads to a growth defect. [PNAS](https://www.pnas.org/doi/10.1073/pnas.2407838121)
Other actin-organising proteins sometimes grouped with crosslinkers:

Abp140 — bundles actin cables but is less well characterised as a canonical crosslinker
Ysc84/Lsb3 — as we discussed, these have bundling activity in vitro but are better described as polymerisation regulators

**Comment by pgaudet** [comment]
Maybe this text in the previous comment should so in the Annotation Review ticket instead ?
